### DISCUSSION TITLE
Uncommitted textbox changes in the Inspector (for example, Transform X or Y position) should be committed on running game

### DISCUSSION THREAD

**Discussion by aspragg** [godotengine/godot-proposals#8734] [Editor]
Create a scene with a Sprite2D at position 100, 100.
Set the scene as the startup scene, and run the project.
The sprite is at 100, 100.
Stop the game.
Edit the X position, setting it to 200, but don't hit Enter or click away.
Press F5 to run the game.
The sprite is still at 100,100 because the edit hasn't been committed.

I propose that hitting F5 to run the project should commit and sweep in the current textbox update.  This is exactly the same behavior as when I edit a script (I don't have to save the script before hitting F5.  I just run the project and my unsaved edits are saved and swept up).

What do you think?

**Comment by theraot** [comment]
Sorry, I misunderstood, this is not about saving the scene, this is about making effective changes from the inspector.

**Comment by Mickeon** [comment]
I don't see any harm in this. If anything, the way it is now could be classified as a bug or odd quirk at best.
